autoclicker.py

```diff
@@ -26,15 +35,17 @@ def on_press(key):
 
 
 def display_controls():
-    print("// AutoClicker by iSayChris")
+    print("// AutoClicker by iSayChris. Additional Features by Derplime.")
     print("// - Settings: ")
-    print("\t delay = " + str(delay) + ' sec' + '\n')
+    print("\t delay = " + str(delay) + ' sec')
```

Comment:
Say it's static delay

Reply:
Addressed in [8a098de](https://github.com/isaychris/python-autoclicker/pull/8/commits/8a098de6a168325f96ace73b85c213575c76c2b2)